rubric.md

```diff
@@ -0,0 +1,25 @@
+# Rubric
```

Comment:
We should format this so it's clear each line means:
Doesn't meet expectations
Meets expectations
Exceed expectations